test/extension/commandExecutor.test.ts

```diff
@@ -125,5 +125,38 @@ suite("commandExecutor", function() {
                     assert.fail(null, null, "react-native command was not expected to fail");
                 });
         });
+
+        test("getReactNativeVersion should return verson string if there is react-native package in node_modules", (done: MochaDone) => {
+            let commandExecutor: CommandExecutor = new CommandExecutor(path.join(__dirname, "..", "resources", "sampleReactNative022Project"));
+
+            commandExecutor.getReactNativeVersion()
+            .then(version => {
+                assert.equal(version, "^0.22.2");
+            }).done(() => done(), done);
```

Comment:
```suggestion
            });
```

Reply:
Reopen since it is still not resolved

Reply:
It seems that such a behavior courses incorrect test's error handling.